tests/unit/test_config.py

```diff
@@ -395,7 +404,7 @@ def test_package_config_parse_error(raw):
                 "specfile_path": "fedora/package.spec",
                 "synced_files": ["fedora/package.spec"],
                 "jobs": [get_job_config_dict_full()],
-                "something": "stupid",
+                # "something": "stupid",
```

Comment:
@TomasTomecek 
in test
https://github.com/packit-service/packit/blob/7ecf2d6e6fd74e1b842282e3bca5112c6f866b30/tests/unit/test_config.py#L284-L295 
is test case which is checking if exception is raised, if there is unknown field (was not working - was failing because of missing synced_files). I decide to go in way to not accept unknown entries in schema. Please confirm it is ok and i will remove this commented lines